spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -42,5 +42,34 @@
 			<artifactId>spring-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- SI channel adapters -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
```

Comment:
Correct, two ways we can skin this cat, either bundle all SI modules in one spring-cloug-gcp-integration or the former which is have integration as a package within each of the services (storage, pubsub,spanner). I think we would be better off with the first approach, we should make this optional (although we may end up needing it for pubsub, since most cases will be as a gateway or perhaps a binder one day)